### ISSUE TITLE
Bug: Danbooru won't download

### ISSUE THREAD

**Issue by hackbot1337** [nullxception/boorusphere#148]
### Type

Booru parser issues

### Bug description

Danbooru images load but when you down all you get is an error.

### Steps to reproduce

1: open app
2: load danbooru
3: select image
4: try to download
5: ????
6: no profit!

### Android version

13 and 14

### App version

Latest

### Screenshots or videos

NA

### Solution

???

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by obscenelyvague** [comment]
+1
Danbooru derivatives fail too.
Relevant
```io.chaldeaprjkt.boorusphere W System.err : at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
io.chaldeaprjkt.boorusphere W System.err : at androidx.work.Worker$a.run(Unknown Source:2)
io.chaldeaprjkt.boorusphere W System.err : at vn.hunghd.flutterdownloader.DownloadWorker.B(Unknown Source:315)
io.chaldeaprjkt.boorusphere W System.err : at vn.hunghd.flutterdownloader.DownloadWorker.r(Unknown Source:482)
io.chaldeaprjkt.boorusphere W System.err : at java.lang.Thread.run(Thread.java:1012)
io.chaldeaprjkt.boorusphere W System.err : at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
io.chaldeaprjkt.boorusphere W System.err : java.lang.NullPointerException: httpConn.contentType must not be null
io.chaldeaprjkt.boorusphere I WM-WorkerWrapper : Worker result FAILURE for Work [ id=ade882e9-9622-4170-81db-ed9bde026899, tags={ flutter_download_task, vn.hunghd.flutterdownloader.DownloadWorker } ]
```

**Comment by hatimurni** [comment]
Get error in android 11 too. Can't download pict ure

**Comment by CnTripto** [comment]
It seems that danbooru will now reject all http1.1 requests

**Comment by nullxception** [comment]
Fixed on #155
